modules/virt-assign-vgpu-passthrough-to-vm.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-configuring-vgpu-passthrough.adoc
+
+[id="virt-assign-vgpu-passthrough-to-vm_{context}"]
+= Assigning vGPU passthrough to virtual machines
+
+Use the {product-title} web console to assign vGPU passthrough for PCI host devices or mediated hardware devices that your virtual machines uses.
+
+.Prerequisites
+
+* Ensure your cluster and virtual machines are deployed in a bare metal environment. At this time, no other environments are supported.
+
+.Procedure
+
+. Assign the GPU or host hardware devices that you want your cluster and virtual machines to access.
+.. In the {product-title} web console, click *Virtualization > Virtual Machines* from the side menu.
+.. Select the virtual machine to which you want to assign the device.
+.. Click the *Details* tab. The *Hardware Devices* field includes links to add or remove *GPU devices* and *Host devices*. Use the minus icon to remove an existing hardware device. You can only add or remove devices from your virtual machine when it is stopped.
+.. Click the pencil icon and use the pop-up windows to add or remove devices, selecting the appropriate hardware resource names. To display resources that are connected to your cluster, click *Compute → Hardware Devices* from the side menu.
```

Comment:
`To display resources that are connected to your cluster, click *Compute → Hardware Devices* from the side menu.` can it be moved outside of the steps? it seems to interrupt the current flow.

Reply:
I moved this into the NOTE block.